apps/src/clientApi.js

```diff
@@ -1,38 +1,71 @@
-/* global dashboard */
 // TODO(dave): Merge with the client API in /shared.
```

Comment:
Is now a good time to address this TODO, or remove it? 😄

If this is the more complete client library, it might make sense to merge the other libraries (which ones?) into this one.